student.rb

```diff
@@ -0,0 +1,12 @@
+require './person'
+
+class Student < Person
+  def initialize(classroom)
+    super(17, 'student')
```

Comment:
- [ ] Kindly consider adding all the parameters to the constructor, also make sure to add parameters to the parent constructor instead of using specific values. the code should be something like this 👇🏼 
```ruby
  def initialize(age, classroom, name = "Unknown", parent_permission = true)  
    super(age, name, parent_permission)
```